content/en/docs/userguide/load_balancing.md

```diff
@@ -0,0 +1,267 @@
+---
+draft: false
+linktitle: Load Balance
+menu:
+  docs:
+    parent: developer guide
+    weight: 10
+title: Load Balance
+toc: true
+type: docs
+
+---
+
+### Before you begin
+
+####  Install Kmesh
+
+ Please refer [quickstart](https://kmesh.net/en/docs/setup/quickstart/)
+
+#### Deploy the Sample Applications
+
+```shell
+[root@master test]# kubectl apply -f sleep.yaml -n tcp-echo-test
+[root@master test]# kubectl apply -f tcp-echo-services.yaml -n tcp-echo-test
+[root@master test]# kubectl apply -f tcp-echo-v1_v2.yaml -n tcp-echo-test
```

Comment:
Where are these yamls, our target is to make a user try your examples successfully without knowing much